.travis.yml

```diff
@@ -5,10 +5,12 @@ jdk:
   - oraclejdk8
 # do not install anything instead return true via unix command true
 install: true
-script: ./core/files/travis-build.sh
+script:
+  - ./core/files/travis-build.sh
+  - ./web/files/travis-build.sh
```

Comment:
Hmmh, as the web jar is also built in the `core/files` we should either avoid a separate `web/files` script or put the web jar building into the `web/files` I fear :)